### DISCUSSION TITLE
anyone have any consumer of the SQL data?

### DISCUSSION THREAD

**Discussion by hartmark** [curzon01/mqtt2mysql#15] [Q&A] [answered]
I found this great project and it's perfect for me to get the mqtt data from my home automation system.

The next step for me is to take this historic data and plot it so I can get graphs for my data. And I thought that perhaps someone have already one working solution to share in GitHub 😀

I'm going to try write a graphing solution using react and perhaps .net as backend part.

PS. Sorry for creating an issue, didn't know about the discussions tab.

**Comment by curzon01** [comment, marked as answer]
Try [Grafana](https://grafana.com/), one of the fastest way to get graphs from history data. It can access MySQL data too
